src/lib.rs

```diff
@@ -1981,6 +2015,22 @@ mod tests {
             })
     }
 
+    fn finds_all_inner<'a>(c: &'a Client) -> impl Future<Item = (), Error = error::CmdError> + 'a {
+        // go to the Wikipedia frontpage this time
+        c.goto("https://en.wikipedia.org/")
+            .and_then(move |_| c.find_all(Locator::Css("#p-interaction li")))
+            .and_then(move |es| {
+                assert_ne!(es.len(), 0);
```

Comment:
I think I'd prefer a stronger assertion than this. If we want to future-proof the test for changes to wikipedia, maybe check that a few known items are in the list?